src/components/Text/Text.tsx

```diff
@@ -104,35 +104,27 @@ const variants = {
   [TextVariants.context]: contextVariant,
 };
 
-const Text = styled.span.attrs<{ variant: keyof typeof TextVariants }>(
-  (props) => ({
-    ...props,
-    className: cls(CLX_TYPOGRAPHY, props?.className),
-  }),
-)<TextProps>`
+const Text = styled.span.attrs<TextProps>((props) => ({
+  ...props,
+  className: cls(CLX_TYPOGRAPHY, props?.className),
+  size: props.size ?? TextSizes.md,
+  variant: props.variant ?? TextVariants.primary,
+  isBold: props.isBold ?? false,
+}))<TextProps>`
   font-family: inherit;
   font-weight: ${({ isBold }) =>
     isBold ? 'var(--sscds-font-weight-body-strong)' : 'inherit'};
   ${({ size }) => sizes[size]};
   ${({ variant }) => variants[variant]};
 `;
 
-Text.defaultProps = {
-  size: TextSizes.md,
-  variant: TextVariants.primary,
-  isBold: false,
-};
-
 export default Text;
 
 export const Strong = styled(Text).attrs<StrongProps>(() => ({
   as: 'strong',
   isBold: true,
-}))``;
-
-Strong.defaultProps = {
   size: TextSizes.inherit,
```

Comment:
```suggestion
  size: props.size ?? TextSizes.inherit,
```

Reply:
What should I put instead?

Reply:
What would you recommend I use instead?

Reply:
<img width="962" alt="image" src="https://github.com/user-attachments/assets/d2818438-c761-4142-9a72-11563552db37">

Reply:
oh you are right I missed that size is omitted for the Strong